/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/umap/distances.py:1063: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/umap/distances.py:1071: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/umap/distances.py:1086: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/umap/umap_.py:660: NumbaDeprecationWarning: [1mThe 'nopython' keyword argument was not supplied to the 'numba.jit' decorator. The implicit default value for this argument is currently False, but it will be changed to True in Numba 0.59.0. See https://numba.readthedocs.io/en/stable/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit for details.[0m
  @numba.jit()
2023/06/21 09:50:07 INFO mlflow.tracking.fluent: Experiment with name 'xenium_human_breast_cancer_one-hop-attention_reference' does not exist. Creating a new experiment.
Traceback (most recent call last):
  File "/home/aih/sebastian.birk/workspace/projects/nichecompass-reproducibility/scripts/reference/../train_nichecompass_reference_model.py", line 351, in <module>
    experiment = mlflow.set_experiment(f"{args.dataset}_{args.model_label}")
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/tracking/fluent.py", line 123, in set_experiment
    experiment_id = client.create_experiment(experiment_name)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/tracking/client.py", line 616, in create_experiment
    return self._tracking_client.create_experiment(name, artifact_location, tags)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/tracking/_tracking_service/client.py", line 240, in create_experiment
    return self.store.create_experiment(
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/store/tracking/file_store.py", line 380, in create_experiment
    return self._create_experiment_with_id(name, str(experiment_id), artifact_location, tags)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/store/tracking/file_store.py", line 345, in _create_experiment_with_id
    write_yaml(meta_dir, FileStore.META_DATA_FILE_NAME, experiment_dict)
  File "/home/aih/sebastian.birk/miniconda3/envs/nichecompass/lib/python3.9/site-packages/mlflow/utils/file_utils.py", line 149, in write_yaml
    raise Exception("Yaml file '%s' exists as '%s" % (file_path, yaml_file_name))
Exception: Yaml file '/home/aih/sebastian.birk/workspace/projects/nichecompass-reproducibility/scripts/reference/mlruns/15/meta.yaml' exists as '/home/aih/sebastian.birk/workspace/projects/nichecompass-reproducibility/scripts/reference/mlruns/15/meta.yaml
